docs/swagger.yaml

```diff
@@ -0,0 +1,978 @@
+        required: true
+      responses:
+        "200":
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/TrainJobStatus"
+        "400":
+          description: "Invalid Params"
+  
+  /evaluation:
+    post:
+      tags:
+      - "Evaluate"
+      summary: "Evaluate the ML model"
+      description: "Evaluate the ML model"
+      operationId: "evaluate_ml_model"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "ml_model_id"
+        in: "query"
+        type: "integer"
+        description: "ML Model Id"
+        required: true
+      - name: "evaluation_dataset_id"
+        in: "query"
+        type: "integer"
+        description: "evaluation_dataset_id"
+      responses:
+        "200":
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/EvaluationJobInfo"
+        "400":
+          description: "invalid params"
+          
+    get:
+      tags:
+      - "Evaluate"
+      summary: "Get the Evaluation Result"
+      description: "Get the Evaluation Result"
+      operationId: "get_evaluation_res"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "evaluation_job_id"
+        in: "query"
+        type: "integer"
+        description: "Evaluation Job ID"
+        required: true
+      responses:
+        "200":
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/EvaluationJobResult"
+        "400":
+          description: "invalid param"
+          
+  /prediction:
+    post:
+      tags:
+      - "Predict"
+      summary: "Prediction by the ML model"
+      description: "Prediction by the ML model"
+      operationId: "predict_by_ml_model"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "ml_model_id"
+        in: "query"
+        type: "integer"
+        description: "ML Model Id"
+        required: true
+      - name: "prediction_dataset_id"
+        in: "query"
+        type: "integer"
+        description: "prediction_dataset_id"
+      responses:
+        "200":
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/PredictionJobInfo"
+        "400":
+          description: "invalid params"
+          
+    get:
+      tags:
+      - "Predict"
+      summary: "Get the Prediction Result"
+      description: "Get the Prediction Result"
+      operationId: "get_prediction_res"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "prediction_job_id"
+        in: "query"
+        type: "integer"
+        description: "Prediction Job ID"
+        required: true
+      responses:
+        "200":
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/PredictionJobResult"
+        "400":
+          description: "invalid param"
+        
+  # /pet/{petId}:
```

Comment:
Just curious, what are these pet related APIs? Does the "pet" word have some special meaning in ML area? Or are we letting users having virtual pets in our website?

If they are commented non-related information, please remove.

Reply:
> Just curious, what are these pet related APIs? Does the "pet" word have some special meaning in ML area? Or are we letting users having virtual pets in our website?
> 
> If they are commented non-related information, please remove.

ShuyuanCao told me that the commented block is the original template, we can remove it once we don't need to use it as a reference any more.